hooks/post_gen_project.py

```diff
@@ -83,7 +85,10 @@ def post_gen():
     subprocess.call(["git", "checkout", "-b", git_new_branch])
     subprocess.call(["git", "add", "."])
     subprocess.call(["git", "commit", "-m", "Initial commit"])
-
-
+    
+def code_of_conduct_clean_up():
+    if '{{cookiecutter.code_of_conduct}}' == 'None':
+        remove_file('{{cookiecutter.project_slug}}/CODE_OF_CONDUCT.md')
```

Comment:
```suggestion
        remove_file('CODE_OF_CONDUCT.md')
```

Could you try if this changes would work?